gracc-backup.py

```diff
@@ -0,0 +1,63 @@
+import sys
+import tomllib
+import os
+import subprocess
+import re
+import time
+
+# Should be either raw, transfer, ps-itb, or ps-prod
+id = sys.argv[1]
+
+def backup_archive(file_path, file_name):
+    toml_path = f'/etc/graccarchive/config/gracc-archive-{id}.toml'
+    with open(toml_path, 'rb') as f:
+        toml_text = tomllib.load(f)
+        output_path = toml_text['Directories']['dest']      
+    local_full_path = "file://" + file_path + file_name
+    remote_full_path = output_path + file_name
+    os.environ['X509_USER_CERT'] = '/etc/grid-security/backup-cert/gracc.opensciencegrid.org-cert.pem'
+    os.environ['X509_USER_KEY'] = '/etc/grid-security/backup-cert/gracc.opensciencegrid.org-key.pem'
```

Comment:
Check if these environment variables are already set before setting them.  That way we can override the "default" values by setting the environment variable in the docker command.  Probably best to use the `get` command from dictionary.  So something like:

```python
os.environ.get('X509_USER_CERT', '/etc/grid-security/backup-cert/gracc.opensciencegrid.org-cert.pem')
os.environ.get('X509_USER_KEY', '/etc/grid-security/backup-cert/gracc.opensciencegrid.org-key.pem')
```